src/Elliptic/Systems/SelfForce/Scalar/Tags.hpp

```diff
@@ -0,0 +1,125 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class ComplexDataVector;
+class DataVector;
+/// \endcond
+
+/*!
+ * \ingroup EllipticSystemsGroup
+ * \brief Items related to solving the self force of a scalar charge on a Kerr
+ * background.
+ *
+ * \see ScalarSelfForce::FirstOrderSystem
+ */
+namespace ScalarSelfForce {}
+
+/// Tags for the ScalarSelfForce system.
+namespace ScalarSelfForce::Tags {
+
+/*!
+ * \brief The complex m-mode field $\Psi_m$.
+ *
+ * Defined by the m-mode decomposition of the complex scalar field $\Phi$
+ * [Eq. (2.6) in \cite Osburn:2022bby ]:
+ *
+ * \begin{equation}
+ * \Phi(t,r,\theta,\phi) = \frac{1}{r} \sum_{m=-\infty}^{\infty}
+ *   \Psi_m(r,\theta) \exp{im\Delta\phi(r)} \exp{im(\phi - \Omega t)}
+ * \end{equation}
+ *
+ * where $\Delta\phi(r) = \frac{a}{r_\plus - r_\minus}
+ * \ln(\frac{r-r_\plus}{r-r_\minus})$.
+ */
+struct MMode : db::SimpleTag {
+  using type = Scalar<ComplexDataVector>;
+};
+
+/*!
+ * \brief The factor multiplying the angular derivative in the principal part of
+ * the equations.
+ *
+ * This is the factor $\alpha$ that defines the principal part of the equations
```

Comment:
What is the function form? Maybe state that it is intended to be set by the analytic data?